runtimes/huggingface/mlserver_huggingface/common.py

```diff
@@ -1,13 +1,16 @@
 import os
+import io
+import base64
 import json
 from typing import Optional, Dict
 from distutils.util import strtobool
+from PIL import Image
```

Comment:
Should we add this as a dependency of the HuggingFace runtime? Or is it already a direct dependency of the `transformers` library?

Reply:
Sorry, I forgot to add the dependency to the requirement.  I will fix it now.